cmd/telegraf/agent.conf

```diff
@@ -94,3 +94,11 @@
   ## stateful plugins on termination of Telegraf. If the file exists on start,
   ## the state in the file will be restored for the plugins.
   # statefile = ""
+
+  ## The interval of time at which the state is peristed into the statefile.
+  ## When set to 0, persist interval is disabled.
+  # statefile_persist_interval = "0h"
```

Comment:
```suggestion
  ## When set to 0s, persist interval is disabled.
  ## Example: statefile_persist_interval = "10m"
  # statefile_persist_interval = "0s"
```

Reply:
Should be resolved as of this [commit](https://github.com/influxdata/telegraf/pull/14561/commits/35de14b84db306904d5376dbea7f3f27ece47a95)